Interfaces/GEOS/geos_interface.F90

```diff
@@ -1284,7 +1291,7 @@ SUBROUTINE GEOS_Diagnostics( GC, IMPORT,  EXPORT, Clock, Phase, &
 
              ! Lightning flash rate
              IF ( LFR(I,J) > 0.0 ) THEN
-                IF ( LWI(I,J) == 1 ) THEN
+                IF ( SfcTypeIndex(1) == 1 ) THEN
```

Comment:
Same question here

Reply:
I think `SfcTypeIndex(1)` can just be reduced to `SfcTypeIndex`.  Functionally they are the same.

Reply:
Thanks @yantosca. If there are no objections from @lizziel then I can make this change before merging.

Reply:
I think that's okay. I originally had it as vector. If it ends up being a problem I can take care of it when I test GEOS.

Reply:
Thanks for confirming!